### ISSUE TITLE
Static Content Not Getting Proxy Base

### ISSUE THREAD

**Issue by Cimplex** [pannal/Kitana#28]
Here is my start command:
```/usr/bin/python3 /opt/Kitana/kitana.py --prefix /kitana --behind-proxy True --proxy-base https://plexbox.us```

Here is my nginx config:
```
    location ^~ /kitana/pms_asset {
        expires 1M;
        access_log off;
        add_header Cache-Control "public";
        proxy_pass http://127.0.0.1:31337;
        #include proxy.conf;
        add_header Transport 'dynamic';
    }

    location ^~ /kitana/static/  {
        root /opt/;

        # versioned static rewrite
        rewrite "^(\/kitana\/static\/(css|js|img)\/)(\w+)\.\w{7}\.(\w{2,4})$" $1$3.$4 last;
    }

    location /kitana/ {
        proxy_pass http://127.0.0.1:31337;
        add_header Transport 'dynamic';
        #include proxy.conf;
    }
```

When you look at the source, it doesn't look like Kitana is acknowledging the proxy=true setting. It works on the local machine (because the machine is hosting the server), but obviously it doesn't work remotely. Notice the static content is trying to come from 127.0.0.1.

![kitana-no-static-content](https://user-images.githubusercontent.com/664988/73579128-3291bc80-4447-11ea-9f8b-842fd41ff6f2.png)

I'm probably doing something stupid. It has to be something wrong with the start command?

**Comment by Cimplex** [comment]
One other thing, I didn't realize that the rewrite was case sensitive on linux. To simplify my setup, I have a symbolic link to the lowercase kitana. LOL

So my /opt/ looks like:

```
lrwxrwxrwx  1 root root    6 Jan 31 23:39 kitana -> Kitana
drwxr-xr-x 10 plex plex 4096 Jan 31 23:10 Kitana
```

Now it actually can serve static content, as seen by the fav icons (the first picture wasn't serving them), but it still is acting like it doesn't know about my proxy base.

![kitana-no-static-content-2](https://user-images.githubusercontent.com/664988/73579956-9c12ca80-4449-11ea-9c5f-da3c7d5c720b.png)

**Comment by pannal** [comment]
Can you try uncommenting proxy.conf in your NGINX? It seems like the headers aren't passed properly.

Should not be the case but might well be.

**Comment by pannal** [comment]
Your issue should be fixed with 0.3.1-3

**Comment by Cimplex** [comment]
Thank you so much for the quick fix. I wasn't able to test until now. All is working remotely!
